examples/react-urql/src/App.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+import { NhostReactProvider, SignedIn, SignedOut } from '@nhost/react'
+import { NhostUrqlProvider } from '@nhost/react-urql'
+
+import { ListPrivatePosts } from './components/ListPrivatePosts'
+import { ListPublicPosts } from './components/ListPublicPosts'
+import { ListPublicPostsSubscription } from './components/ListPublicPostsSubscription'
+import { SignIn } from './components/SignIn'
+import { nhost } from './utils/nhost'
+
+function App() {
+  return (
+    <NhostReactProvider nhost={nhost}>
+      <NhostUrqlProvider nhost={nhost}>
+        <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8 mt-6">
+          <div className="mx-auto max-w-3xl">
+            <SignedIn>
```

Comment:
Not relevant to this PR, but now that I see this approach being used, I like this `SignedIn` / `SignedOut` approach even less than I did when we first discussed it. I would most likely never use it like this in the dashboard.

It's a mental overhead when I'm looking at something deep in the component tree to know if that part is relevant for an authenticated or an unauthenticated state.

I think an early exit strategy / a layout-level authentication strategy is much clearer and easier to follow.